server/send_notify_request.py

```diff
@@ -7,17 +7,16 @@
 print "Getting image details from test phase"
 beanstalk_host = sys.argv[1]
 output_image = sys.argv[2]
-notify_email  = sys.argv[3]
 
 msg_details = {}
 msg_details['action'] = "notify_user"
 msg_details['subject'] = "cccp-build is complete"
-msg_details['msg'] = "Build is successful please pull the image ("+output_image+")"
-msg_details['to_mail'] = notify_email
+msg_details['msg'] = ("Build is successful please "
+                      "pull the image (%s) " % output_image)
```

Comment:
Since we're removing `msg_details['to_mail'] = notify_email` here, would this fail? Asking sinc we're dumping `msg_details` variable on the `master_tube`. So, the job that's being put on the tube won't have the email details.